src/cluster_legacy.h

```diff
@@ -305,6 +305,7 @@ struct _clusterNode {
     clusterLink *link;          /* TCP/IP link established toward this node */
     clusterLink *inbound_link;  /* TCP/IP link accepted from this node */
     list *fail_reports;         /* List of nodes signaling this as failing */
+    uint16_t extensions_supported; /* Does the node support parsing ping extensions. */
```

Comment:
```suggestion
    bool extensions_supported; /* Does the node support parsing ping extensions. */
```

Reply:
On a second thought, should we make this a flag instead?

Reply:
Yes, on the redis issue I suggested it be added into the flags. That also makes it automatically get propagated in the gossip.